rest/health.go

```diff
@@ -41,17 +41,12 @@ type HealthResponse struct {
 	Status HealthStatus `json:"status"`
 	// Message gives additional information on the overall health state.
 	Message string `json:"message"`
-	// Services is map of services by name with their individual health results.
-	Services map[string]HealthResult `json:"services"`
+	// provides further information on the result e.g. services or partitions
+	Information map[string]HealthResult `json:"information"`
```

Comment:
I think you can keep the original name to prevent a breaking API change for the clients. `services` is still an appropriate name. Also (by Go convention) doc strings need to be sentences that start with the element they describe.

```suggestion
	// Services contain the individual health results of the services as evaluated by the HealthCheck interface. The overall HealthStatus is then derived automatically from the results of the health checks. 
	// 
	// Note that the individual HealthResults evaluated by the HealthCheck interface may again consist of a plurality services. While this is only optional it allows for creating nested health structures. These can be used for more sophisticated scenarios like evaluating platform health describing service availability in different locations or similar.
	//
	// If using nested HealthResults, the status of the parent service can be derived automatically from the status of its children by leaving the parent's health status field blank. 
	Services map[string]HealthResult `json:"services"`
```